### DISCUSSION TITLE
Add `$rowClickable` property to resource

### DISCUSSION THREAD

**Discussion by hivokas** [laravel/nova-issues#4233] [Q&A] [answered]
Sometimes I want rows on resource index to be not clickable. 

Currently the only way to prevent clicking on row is to unauthorize admins from viewing the resource:

<img width="529" alt="image" src="https://user-images.githubusercontent.com/22997803/168276317-332f9054-a079-4340-8888-a05259780432.png">

It would be great to have `$rowClickable` or something like that on resource to have an ability to prevent opening the resource when clicking on the row on resource index.

**Comment by LTKort** [comment, marked as answer]
This is the same as #4215 right?

**Reply by hivokas** [reply]
Yeah I think the idea is very similar
